From 2e1be0f31f2c5c57a6004ca6630f90fe56564775 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Sun, 30 Jan 2022 23:46:37 +0100
Subject: [PATCH 365/389] usb: typec: fusb302: Slightly increase wait time for
 BC1.2 result

This is not strictly necessary, but it should not hurt. Looking at
kernel log timing, the BC1.2 detection completion time and this
timeout are quite tight. This adds some headspace just in case.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/usb/typec/tcpm/fusb302.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/usb/typec/tcpm/fusb302.c b/drivers/usb/typec/tcpm/fusb302.c
index 721b2a548084..1f12e8432f12 100644
--- a/drivers/usb/typec/tcpm/fusb302.c
+++ b/drivers/usb/typec/tcpm/fusb302.c
@@ -445,9 +445,9 @@ static int tcpm_get_current_limit(struct tcpc_dev *dev)
 
 	/*
 	 * USB2 Charger detection may still be in progress when we get here,
-	 * this can take upto 600ms, wait 800ms max.
+	 * this can take upto 600ms, wait 1000ms max.
 	 */
-	timeout = jiffies + msecs_to_jiffies(800);
+	timeout = jiffies + msecs_to_jiffies(1000);
 	do {
 		if (extcon_get_state(chip->extcon, EXTCON_CHG_USB_SDP) == 1)
 			current_limit = 500;
-- 
2.35.3

